source/clear-linux/guides/telemetrics/telemetrics.rst

```diff
@@ -0,0 +1,19 @@
+.. _telemetrics:
+
+Telemetrics in |CL-ATTR|
```

Comment:
Please revise page title to simply identify the topic and not use the product name e.g. "Telemetrics"